sensu/plugins/check-neutron-agents-duplicate.py

```diff
@@ -71,7 +71,7 @@ def main():
     args = parser.parse_args()
 
     CRITICALITY = args.criticality
-    url = args.auth_url + '/tokens'
+    url = args.auth_url.rstrip('5000/v3') + '35357/v2.0/tokens'
```

Comment:
If the auth_url port/api_version were to change this check would break.  Also it would be better to be able to override these values externally and inject them.

Maybe we could add support for an auth_port and auth_api_version and do something like this ?

```
# assumes an auth_url of the format https://blahblahblah:PORT/API_VERSION
pieces = args.auth_url.split(':')
protocol = pieces[0]
url = pieces[1]
_pieces = pieces[2].split('/')
port = _pieces[0]
api_version = _pieces[1]

if args.auth_port:
    url = args.auth_port

if args.auth_api_version:
    api_version = args.auth_api_version

url = protocol + ":" + auth_url + ':' + port + "/" + api_version + "/tokens"
```